flashlight/pkg/CMakeLists.txt

```diff
@@ -1,18 +1,36 @@
 cmake_minimum_required(VERSION 3.10)
 
-set(FL_PKG_VISION_DIR ${FL_PKG_DIR}/vision)
-include(${FL_PKG_VISION_DIR}/CMakeLists.txt)
+if(FL_BUILD_PKG_RUNTIME)
+  set(FL_PKG_RUNTIME_DIR ${FL_PKG_DIR}/runtime)
+  include(${FL_PKG_RUNTIME_DIR}/CMakeLists.txt)
+  set(INSTALLABLE_TARGETS ${INSTALLABLE_TARGETS} fl_pkg_runtime)
+  setup_install_headers(${FL_PKG_RUNTIME_DIR} ${FL_INSTALL_INC_DIR_HEADER_LOC}/pkg)
```

Comment:
Have you checked that the layout of the installed headers is correct given these changes? Probably fine, but worth checking

Reply:
Checked! Looks good to me.